### PR TITLE
coordinator feature

### PR DESCRIPTION
**Pull request by weizhoublue**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

File: charts/spiderpool/values.yaml [hunk 1]
```diff
@@ -49,6 +49,28 @@ feature:
   ## @param feature.enableSpiderSubnet SpiderSubnet feature gate.
   enableSpiderSubnet: true
 
+  coordinator:
+    ## @param feature.coordinator.enabled enable SpiderCoordinator
+    enabled: false
```
Comment (weizhoublue) [inline charts/spiderpool/values.yaml:54]: default to true

**Comment by weizhoublue** [comment]
<https://github.com/spidernet-io/spiderpool/issues/1831>

File: charts/spiderpool/values.yaml (hunk 1 above)
Comment (cyclinder) [reply charts/spiderpool/values.yaml]: 这个打开会导致chart lint 失败:

https://github.com/spidernet-io/spiderpool/actions/runs/5116428213/jobs/9199126749

Comment (cyclinder) [reply charts/spiderpool/values.yaml:54]: 而且alpha的feature开关一般默认是关闭，beta和ga之后才会默认开启： https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

**Pull request merged**